### DISCUSSION TITLE
Help with non branded power metering plug

### DISCUSSION THREAD

**Discussion by pzoltan92** [arendst/Tasmota#15180] [Q&A] [answered]
Hello! I have successfully installed Tasmota on my Non Branded power plug. Previously I was using this  with tuya. Now after converting it, I could find some templates based on devices which looks similar. In spite of this I can not get it working. It turns on the LED and I can hear a clicking sound, but no devies are powered plugged into this socket. Before tasmotizing it was running without any issues. Strange is that is outputs the voltage, but the other values are 0. I have tried to mess around with settings but still no success. What can I do now? 
For reference I have used:
https://templates.blakadder.com/framexx_FXIOWR16.html
Here is a picture of my device. No marking on the frame of it so I can not figure out the manufacturer.
![IMG-1728](https://user-images.githubusercontent.com/73195065/159132580-c2a05655-12dc-4588-b5db-beee678260d5.jpg)

Any idea is appreciated!

**Comment by barbudor** [comment, marked as answer]
How did you converted it ? Did you openned it for serial flashing ? If yes, that was a good opportunity to checks the connections between the ESP and the different pieces by tracking the signals on hte PCB.

Did you get the button to work ?
If not you could start by not configuring the power measurment but identify button relay and leds using https://tasmota.github.io/docs/Configuration-Procedure-for-New-Devices/

**Reply by sfromis** [reply]
There are decent chances that it is the same as one of the other physically similar devices on https://templates.blakadder.com/plug.html - as such devices are frequently rebranded, sometimes without worrying about paying for a new brand name on the physical device/packaging.

You may also find inspiration looking at those template, to see how it's usually done. While it is possible that the gpio pins are not exactly the same, it is likely one of the already supported energy monitoring chip setups.
